cypress/integration/home.spec.js

```diff
@@ -1,12 +1,19 @@
+import terminalLog from '../plugins/terminalLog'
+
 /* eslint-disable no-undef */
 /// <reference types="Cypress" />
 
 describe("home page", () => {
     beforeEach(() => {
         cy.visit("/");
+        cy.injectAxe();
     });
 
     it("landing page loaded", () =>{
         cy.url().should("contains", "/");
     })
+
+    it('Has no detectable a11y violations on load', () => {
+        cy.checkA11y(null, null, terminalLog)
```

Comment:
Seems like a weird function signature with the two `null`s but it's in the example, so 🤷

Reply:
Yeah not the prettiest signature...